src/server/deamon/index.ts

```diff
@@ -155,12 +178,27 @@ export class DaemonServer extends Dispose {
     notification.show(params['message']);
   };
 
-  private connected = () => {
+  private connected = async () => {
     statusBar.updateDevice(undefined, true);
-    this.sendRequest({
-      id: this.currentId++,
-      method: 'device.enable',
-    });
+    try {
+      await this.sendRequest({
+        id: this.currentId++,
+        method: 'device.enable',
+      });
+      // wait 10 seconds
+      await delay(10000);
```

Comment:
The delay promise is a nice touch! But let’s use more than 10 seconds since flutter internally uses a 10 second timeout for getting the devices

Reply:
Done